### DISCUSSION TITLE
Craftist_0.7.1

### DISCUSSION THREAD

**Discussion by FlohGro-dev** [FlohGro-dev/Craftist#49] [Announcements]
### v0.7.1

- **fixed:** settings for continuous sync and due dates in daily notes were not read properly
- **improved:** notification for unsynced tasks (recurring tasks are not synced when due dates are disabled)

<hr /><em>This discussion was created from the release <a href='https://github.com/FlohGro-dev/Craftist/releases/tag/0.7.1'>Craftist_0.7.1</a>.</em>

**Comment by ThreeGK** [comment]
It may be user error but I am having two major issues:

1. The API Key does not "stick" - it works initially but then the next time I am in Craft, I have to add it again.
2. I cannot get "Today's Tasks" download to work in the web version of Craft. Everything else seems to work fine.

**Reply by FlohGro-dev** [reply]
Thanks for letting me know.

Can you please double check that you have the newest version installed? Maybe remove it and install it again.
If the issue persists please provide a few more details so I can reproduce it (e.g. the platform you're using it, what steps you did, error messages presented by the eXtension:)

**Reply by ThreeGK** [reply]
I uninstalled and reinstalled and the today tasks portion works.
I will look at the API key over the next day and see what happens.

**Reply by FlohGro-dev** [reply]
Ok great to hear!
Let me know if the API key gets lost again and if possible how to reproduce this.

If you're in the web make sure to let docs.craft.do store data in your browser
